apps/web-giddh/src/app/theme/tax-authority/reports/account-wise-report/account-wise-report.component.html

```diff
@@ -117,18 +117,20 @@
                                             ></amount-field>
                                         </td>
                                     </ng-container>
-        
+
                                     <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
                                     <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
                                     <tr mat-footer-row *matFooterRowDef="displayedColumns"></tr>
                                 </table>
                             </div>
                         </div>
                         <!-- No Data Found Message -->
-                        <div class="no-data" *ngIf="(accountWiseReport$ | async)?.responses?.length === 0 && !(isLoading$ | async) && !isTaxApiInProgress">
```

Comment:
why you are removing this condition ?

Reply:
(accountWiseReport$ | async)?.responses?.length === 0  
This condition will be true if I have responses, but not responses if I don't have a tax number.
!(isLoading$ | async) 
this condition already checked by the parent div.
!isTaxApiInProgress
this condition already checked by the parent div